plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -674,6 +679,8 @@ if( options.count(op_name) ) { \
 
 void producer_plugin::plugin_initialize(const boost::program_options::variables_map& options)
 { try {
+   my->blockvault_plug = app().find_plugin<blockvault_client_plugin>();
+   EOS_ASSERT( my->blockvault_plug, plugin_config_exception, "blockvault_client_plugin not found" );
```

Comment:
Once blockvault_plugin removed from main this will need to be removed and check for null at use.